### PR TITLE
Resolve conflicts

### PR DESCRIPTION
**Pull request by peifeng-unity**

### CODE REVIEW

File: com.unity.robotics.urdf-importer/Editor/NeedsRuntimeConversion/Extensions/UrdfRobotExtensions.cs [hunk 1]
```diff
@@ -54,6 +54,8 @@ public static IEnumerator<GameObject> Create(string filename, ImportSettings set
             Robot robot = new Robot(filename);
             settings.totalLinks = robot.links.Count;
 
+            settings.totalLinks = robot.links.Count;
```
Comment (hyounesy) [inline com.unity.robotics.urdf-importer/Editor/NeedsRuntimeConversion/Extensions/UrdfRobotExtensions.cs:57]: This is a duplicate line?

File: com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs [hunk 2]
```diff
@@ -12,27 +12,13 @@ public override void OnInspectorGUI()
...
-                {
-                    rigidbody.inertiaTensorRotation = Quaternion.Euler(newEuler);
-                }
-            }
+            Quaternion inertiaTensorRotation = new Quaternion();
+            inertiaTensorRotation.eulerAngles = EditorGUILayout.Vector3Field("Inertia Tensor Rotation", _rigidbody.inertiaTensorRotation.eulerAngles);
+            _rigidbody.inertiaTensorRotation = inertiaTensorRotation;
         }
     }
 }
```
Comment (hyounesy) [inline com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs:24]: add new line before eof

File: com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs [hunk 3]
```diff
@@ -12,27 +12,13 @@ public override void OnInspectorGUI()
...
-                Vector3 inertiaTensorEuler = rigidbody.inertiaTensorRotation.eulerAngles;
-                Vector3 newEuler = EditorGUILayout.Vector3Field("Inertia Tensor Rotation", inertiaTensorEuler);
-                if (inertiaTensorEuler != newEuler)
-                {
-                    rigidbody.inertiaTensorRotation = Quaternion.Euler(newEuler);
-                }
-            }
+            Quaternion inertiaTensorRotation = new Quaternion();
+            inertiaTensorRotation.eulerAngles = EditorGUILayout.Vector3Field("Inertia Tensor Rotation", _rigidbody.inertiaTensorRotation.eulerAngles);
+            _rigidbody.inertiaTensorRotation = inertiaTensorRotation;
```
Comment (hyounesy) [inline com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs:21]: Is this entire code block intentionally removed?

**Review by hyounesy — CHANGES_REQUESTED** [review]
I am not sure about the code removed in RigidbodyEditor. Is this ok @vidurvij-Unity?

File: com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs (hunk 3 above)
Comment (peifeng-unity) [reply com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs:21]: This is what I don't know it is removed automatically when merging main to dev

File: com.unity.robotics.urdf-importer/Editor/NeedsRuntimeConversion/Extensions/UrdfRobotExtensions.cs (hunk 1 above)
Comment (peifeng-unity) [reply com.unity.robotics.urdf-importer/Editor/NeedsRuntimeConversion/Extensions/UrdfRobotExtensions.cs]: Yeah, fixed.

File: com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs (hunk 2 above)
Comment (peifeng-unity) [reply com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs]: done!

File: com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs (hunk 3 above)
Comment (vidurvij-Unity) [reply com.unity.robotics.urdf-importer/Editor/CustomEditors/RigidbodyEditor.cs:19]: This should be removed. This was changed against this issue.
https://github.com/Unity-Technologies/URDF-Importer/issues/59

**Review by vidurvij-Unity — APPROVED** [review]

**Review by hyounesy — APPROVED** [review]

**Pull request merged**